pages/common/bgpgrep.md

```diff
@@ -0,0 +1,29 @@
+# bgpgrep
+
+> Filter and print BGP data within MRT dumps.
+> Can read files compressed with gzip, bzip2 and xz.
+> More information: <https://gitea.it/1414codeforge/ubgpsuite>.
+
+- Output all routes:
+
+`bgpgrep {{master6.mrt}}`
+
+- Output all routes received from a specific peer, determined by the AS number:
```

Comment:
Is "determined" appropriate here? What is meant is the following: In a MRT file are routes, which were received from different peers. This command shows all routes which were received from the peer with the ASN 64498.